package.json

```diff
@@ -71,35 +71,35 @@
     "website:preview": "lite-server --baseDir=dist/website/browser"
   },
   "dependencies": {
-    "@angular/animations": "8.0.0-beta.5",
-    "@angular/common": "8.0.0-beta.5",
-    "@angular/compiler": "8.0.0-beta.5",
-    "@angular/core": "8.0.0-beta.5",
-    "@angular/forms": "8.0.0-beta.5",
-    "@angular/http": "8.0.0-beta.5",
-    "@angular/platform-browser": "8.0.0-beta.5",
-    "@angular/platform-browser-dynamic": "8.0.0-beta.5",
-    "@angular/platform-server": "8.0.0-beta.5",
-    "@angular/router": "8.0.0-beta.5",
+    "@angular/animations": "8.0.0-beta.13",
+    "@angular/common": "8.0.0-beta.13",
+    "@angular/compiler": "8.0.0-beta.13",
+    "@angular/core": "8.0.0-beta.13",
+    "@angular/forms": "8.0.0-beta.13",
+    "@angular/http": "8.0.0-beta.10",
+    "@angular/platform-browser": "8.0.0-beta.13",
+    "@angular/platform-browser-dynamic": "8.0.0-beta.13",
+    "@angular/platform-server": "8.0.0-beta.13",
+    "@angular/router": "8.0.0-beta.13",
     "@nguniversal/express-engine": "7.0.2",
     "@nguniversal/module-map-ngfactory-loader": "7.0.2",
     "@webcomponents/custom-elements": "1.1.0",
     "core-js": "2.5.5",
     "express": "4.16.4",
     "jump.js": "1.0.2",
     "prismjs": "1.14.0",
-    "rxjs": "6.3.3",
+    "rxjs": "6.4.0",
     "web-animations-js": "2.3.1",
     "zone.js": "0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.14.0-beta.5",
-    "@angular-devkit/build-ng-packagr": "0.14.0-beta.5",
-    "@angular-devkit/build-optimizer": "0.14.0-beta.5",
-    "@angular-devkit/core": "8.0.0-beta.5",
-    "@angular/cli": "8.0.0-beta.5",
-    "@angular/compiler-cli": "8.0.0-beta.5",
-    "@angular/language-service": "8.0.0-beta.5",
+    "@angular-devkit/build-angular": "0.800.0-beta.15",
+    "@angular-devkit/build-ng-packagr": "0.800.0-beta.15",
+    "@angular-devkit/build-optimizer": "0.800.0-beta.15",
+    "@angular-devkit/core": "8.0.0-beta.15",
+    "@angular/cli": "8.0.0-beta.15",
+    "@angular/compiler-cli": "8.0.0-beta.13",
```

Comment:
It's not `8.0.0-beta.15`?

Reply:
no, not as of when I was updating them today.